suitcase/tests/conftest.py

```diff
@@ -1,30 +1,43 @@
 import uuid
 import pytest
-from metadatastore.mds import MDS
-from databroker.broker import Broker
+from databroker import Broker
 import os
 
 AUTH = os.environ.get('MDSTESTWITHAUTH', False)
 
 
+test_config = {
+    'metadatastore': {
+        'module': 'databroker.headersource.mongo',
+        'class': 'MDS',
+        'config': {
+            'host': 'localhost',
+            'port': 27017,
+            'database': 'test1',
+            'timezone': 'US/Eastern'}
+    },
+    'assets': {
+        'module': 'databroker.assets.mongo',
+        'class': 'Registry',
+        'config': {
+            'host': 'localhost',
+            'port': 27017,
+            'database': 'test2'}
+    }
+}
+
+
 @pytest.fixture(params=[1], scope='function')
 def db_all(request):
     '''Provide a function level scoped metadatastore instance talking to
     temporary database on localhost:27017 with focus on v1.
     '''
-    db_name = "mds_testing_disposable_{}".format(str(uuid.uuid4()))
-    test_conf = dict(database=db_name, host='localhost',
-                     port=27017, timezone='US/Eastern',
-                     mongo_user='tom',
-                     mongo_pwd='jerry')
-    #version_v = request.param
-    mds = MDS(test_conf, auth=AUTH)
-
-    db = Broker(mds, fs=None)
+    db = Broker.from_config(test_config)
```

Comment:
It is better to throw a uid into the database name so that you are sure you get a clean database and are not stepping on anyone's toes.

Reply:
good point.

Reply:
There are a couple differences here. Notice that the old one created a new, clean database each time. This one would reuse the same database each time, potentially introducing coupling between the tests. Also, the old name was not _just_ a UUID: it started with 'mds_testing_disposable_' so that someone looking at a list of mongo databases has a clue where they came from.

Reply:
Do you plan to address this?

Reply:
Can you please address this and generate a fresh database every time?

Reply:
Re-upping this.